site/reference/cli/alpha/repo/get/README.md

```diff
@@ -0,0 +1,48 @@
+---
+title: "`get`"
+linkTitle: "get"
+type: docs
+description: >
+  List registered repositories.
+---
+
+<!--mdtogo:Short
+    List registered repositories.
+-->
+
+`get` lists registered repositories.
```

Comment:
I am thinking if we should rename this to `list` :) 

`kpt alpha repo list` ?

Reply:
`kubectl` uses `get` though :(
also, `list` is an alias in place for `kpt alpha repo get` (https://github.com/GoogleContainerTools/kpt/blob/main/internal/cmdrepoget/command.go#L52)

Reply:
Good to know that we have an alias :)

For some reason, my brain associates `get` with fetching single entity, but it could be just me. list alias solves my problem then so will stop bikeshedding :).